ora2/jobs/AddCourseToSandbox.groovy

```diff
@@ -0,0 +1,40 @@
+/*
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    * JOB_FOLDER_NAME: some-folder
+    * SSH_CREDENTIAL_NAME : your-ssh-credential-name
+	* SSH_USERNAME : your-ssh-username
```

Comment:
Ah right on, looks like hard tabs vs soft tabs here, can fix that and the formatting below.